docs/content/docs/install/_index.md

```diff
@@ -43,16 +56,29 @@ $ osm install
 OSM installed successfully in namespace [osm-system] with mesh name [osm]
 ```
 
-By default, the control plane components are installed into a Kubernetes Namespace called `osm-system` and the control plane is given a unique identifier attribute `mesh-name` defaulted to `osm`. Both the Namespace and mesh-name can be configured with flags to the `osm install` command. Running `osm install --help` provides details on the various flags that can be configured.
+Run `osm install --help` for more options.
 
-The `mesh-name` is a unique identifier assigned to an osm-controller instance during install to identify and manage a mesh instance.
+### Using the Helm CLI
```

Comment:
I would add a line here stating that the Helm chart is located at github.com/openservicemesh/osm/charts/osm